### PR TITLE
Add NOTICE-file and generator

### PR DESCRIPTION
**Pull request by bkcsoft**
See title

@tboerger I think you have to create the .sig-file for Drone

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,6 +63,14 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
+NOTICE: vendor/
+	rm -f NOTICE # `govendor license` appends...
+	govendor license -o NOTICE +v
+
+notice-up-to-date: NOTICE
```
Comment (sapk) [inline Makefile:79]: Since it is a test maybe prepend it name with test- ? and add .PHONY just in case ?

Comment (sapk) [reply Makefile:79]: Maybe we could even used a more generic name like test-generate-diff and include generation of NOTICE in make generate or in go generate ?

**Comment by tboerger** [comment]
I don't think that it makes sense to list all license files in detail. Beside that it's not properly formatted. A listing of the used licenses? Ok... But every license? Wtf?

**Comment by bkcsoft** [comment]
@tboerger It lists all vendored licenses. Anything that isn't vendored shouldn't be in use and vice versa :)

**Comment by bkcsoft** [comment]
@tboerger This is standard practice for larger projects though :/

**Comment by strk** [comment]
Why adding the NOTICE file at all if it can be generated from vendor/ directory ?

**Pull request closed without merging**